packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -71,326 +36,415 @@
-                                ></span>
+                                <div class="oui-tile__description">
+                                    <span
+                                        class="d-block mb-2"
+                                        data-ng-if="!$ctrl.device.editMode"
+                                        data-ng-bind="$ctrl.device.deviceId"
+                                    >
+                                    </span>
+                                    <button
+                                        class="btn btn-default no-transition"
+                                        data-translate="overTheBox_deviceId_modify"
+                                        type="button"
+                                        title="{{::'overTheBox_deviceId_button_title' | translate }} {{::'overTheBox_deviceId_modify' | translate }}"
+                                        data-ng-if="!$ctrl.device.editMode && $ctrl.device.deviceId"
+                                        data-ng-click="$ctrl.device.editMode = true; $ctrl.deviceIdToLink = $ctrl.device.deviceId"
+                                    ></button>
+                                    <form
+                                        class="form-inline"
+                                        name="formEditDevice"
+                                        novalidate
+                                        data-ng-if="$ctrl.device.editMode || !$ctrl.device.deviceId"
+                                    >
+                                        <div
+                                            class="form-group"
+                                            data-ng-class="{
+                                                        'has-error': formEditDevice.inputInlineDeviceId.$touched && formEditDevice.inputInlineDeviceId.$invalid
+                                                    }"
+                                        >
+                                            <label
+                                                for="inputInlineDeviceId"
+                                                class="sr-only"
+                                                data-translate="overTheBox_deviceId"
+                                            >
+                                            </label>
+
+                                            <!-- list des items : $ctrl.deviceIds -->
+                                            <input
+                                                class="form-control mr-2"
+                                                id="inputInlineDeviceId"
+                                                name="inputInlineDeviceId"
+                                                data-typeahead="deviceId for deviceId in $ctrl.deviceIds"
+                                                data-ng-pattern="/[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}/"
```

Comment:
```suggestion
                                                data-ng-pattern="$ctrl.MY_SUPER_REGEX"
```
Move the regex in constant file